source/type/pb_converter.cpp

```diff
@@ -593,6 +594,49 @@ void PbCreator::Filler::fill_pb_object(const ng::Admin* adm, pbnavitia::Administ
     }
 }
 
+void PbCreator::Filler::fill_access_points(const std::set<nt::AccessPoint*> access_points,
```

Comment:
```suggestion
void PbCreator::Filler::fill_access_points(const std::set<nt::AccessPoint*> & access_points,
```

Reply:
just a small remark, since your access_point are sorted with their memory address(std::set<nt::AccessPoint*>), the outcome maybe random, that could be problematic for your tests